client-tools/Backup/arangobackup.cpp

```diff
@@ -50,6 +51,7 @@ int main(int argc, char* argv[]) {
   TRI_GET_ARGV(argc, argv);
   return ClientFeature::runMain(argc, argv, [&](int argc, char* argv[]) -> int {
     ArangoGlobalContext context(argc, argv, BIN_DIRECTORY);
+    arangodb::signals::maskAllSignalsClient();
     context.installHup();
```

Comment:
Do we really need a handler for SIGHUP in the client tools? I know it was there before already, but I have my doubts this is useful.